torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -757,6 +757,24 @@ std::pair<std::shared_ptr<Operator>, Stack> getOpWithStack(
   }
 }
 
+// This function is used to check if the schema is valid for the given args and
+// kwargs It skips checking script object to allow FakeScriptObject to be
+// matched. It returns the stack if the arugments matches schema and false
+// otherwise.
+Stack checkSchemaAllowFakeScriptObject(
+    const FunctionSchema& schema,
+    py::args args,
+    const py::kwargs& kwargs) {
+  Stack stack;
+  try {
+    stack = createStackForSchema(
+        schema, std::move(args), kwargs, c10::nullopt, true);
```

Comment:
er, what happens if a FakeScript object gets here?

Reply:
We'll check it with fakeScriptObjectToIValue, which will check whether the current FakeScriptObject is an instance of the fake class corresponding to the torch class specified in the schema.